docs/data-sources/metastores.md

```diff
@@ -3,11 +3,11 @@ subcategory: "Unity Catalog"
 ---
 # databricks_metastores Data Source
 
--> **Note** This data source can only be used with an account-level provider!
-
 Retrieves a mapping of name to id of [databricks_metastore](../resources/metastore.md) objects, that were created by Terraform or manually, so that special handling could be applied.
 
--> **Note** [`account_id`](../index.md#account_id) provider configuration property is required for this resource to work. Data resource will error in case of metastores with duplicate names. This data source is only available for users & service principals with account admin status
+-> This data source can only be used with an account-level provider!
+
+-> [`account_id`](../index.md#account_id) provider configuration property is required for this resource to work. Data resource will error in case of metastores with duplicate names. This data source is only available for users & service principals with account admin status
```

Comment:
> This data source is only available for users & service principals with account admin status

this note applies to most of account-level resources right?

Reply:
yes. I'm really not sure that we should have it. I was just moving the comment around